optuna/study.py

```diff
@@ -322,6 +322,19 @@ def _should_stop() -> bool:
     def set_user_attr(self, key: str, value: Any) -> None:
         """Set a user attribute to the study.
 
+        Example:
+
+            import optuna
+
+            def objective(trial):
+                x = trial.suggest_float("x", -10, 10)
+                return (x - 2) ** 2
+
```

Comment:
This objective is not used. How about removing it?

Reply:
Thanks for swift review and nice idea!